extern/sector-storage/manager_test.go

```diff
@@ -45,8 +45,7 @@ func (t testStorage) DiskUsage(path string) (int64, error) {
 }
 
 func newTestStorage(t *testing.T) *testStorage {
-	tp, err := ioutil.TempDir(os.TempDir(), "sector-storage-test-")
-	require.NoError(t, err)
+	tp := t.TempDir()
```

Comment:
This needs to be left as a ioutil temp dir to respect the `LOTUS_TEST_NO_CLEANUP` env var that was introduced for debugging